thirdparty/github.go

```diff
@@ -1881,3 +1881,40 @@ func GetCheckRun(ctx context.Context, owner, repo string, checkRunID int64) (*gi
 	}
 	return checkRun, nil
 }
+func GetCommitAuthorEmail(ctx context.Context, owner, repo string, prNumber int) (string, error) {
```

Comment:
Please add a new line before this function, and a function comment, since this function is exported (always add function comments to public functions in this repo)

Reply:
This isn't the right title anymore also, it should be GetCoAuthorEmail